src/api.ts

```diff
@@ -28,10 +28,7 @@ export class API {
      * Check if a doujin exists
      * @param id ID of the doujin
      */
-    doujinExists(id: number | string): Promise<boolean> {
-        id = Number(id);
-
-        if (isNaN(id)) throw new TypeError('id is not a number');
+    doujinExists(id: number): Promise<boolean> {
```

Comment:
I made methods accepting an ID use `number | string` to reduce duplicated code. Why would you change this?